tests/phpunit/api/v3/ReportTemplateTest.php

```diff
@@ -364,7 +364,7 @@ public function testLybuntReportWithFYData() {
       SELECT SQL_CALC_FOUND_ROWS contact_civireport.id as cid  FROM civicrm_contact contact_civireport    INNER JOIN civicrm_contribution contribution_civireport USE index (received_date) ON contribution_civireport.contact_id = contact_civireport.id
          AND contribution_civireport.is_test = 0
          AND contribution_civireport.receive_date BETWEEN \'20140701000000\' AND \'20150630235959\'
-       
+
```

Comment:
The failing test is due to this whitespace removal by my editor 🤦‍♂️ . Will update the PR with the fix.